src/test/datascience/notebook/helper.ts

```diff
@@ -383,14 +382,12 @@ export async function waitForKernelToGetAutoSelected(expectedLanguage?: string,
     return waitForKernelToChange(criteria, timeout);
 }
 
-export async function startJupyterServer(api?: IExtensionTestApi) {
-    const { serviceContainer } = api ? { serviceContainer: api.serviceContainer } : await getServices();
+export async function startJupyterServer(notebook?: NotebookDocument) {
     if (IS_REMOTE_NATIVE_TEST) {
-        const selector = serviceContainer.get<JupyterServerSelector>(JupyterServerSelector);
         const uri = await JupyterServer.instance.startJupyterWithToken();
         const uriString = decodeURIComponent(uri.toString());
         traceInfo(`Jupyter started and listening at ${uriString}`);
-        await selector.setJupyterURIToRemote(uriString);
+        return commands.executeCommand('jupyter.selectjupyteruri', false, uri, notebook);
```

Comment:
@rebornix this is how you'd call the command. 

Arguments are:
- arg[1] = boolean indicating local or remote. Set to false
- arg[2] = URI of the jupyter server
- arg[3] = NotebookDocument of what you want a preferred kernel for